readme.md

```diff
@@ -39,20 +39,30 @@ Fingerprint Pro Plus expands on Fingerprint Pro’s identification via the visit
 
 #### Smart Signals 
 
-Smart Signals are an extensive set of signals, not used in traditional identification processes, that are used to identify anonymous visitors attempting to perform fraudulent activities enabling you to identify them with confidence while providing a frictionless, positive experience for trusted visitors. 
+[Smart Signals](https://dev.fingerprint.com/docs/smart-signals-overview) are an extensive set of signals, not used in traditional identification processes, that are used to identify anonymous visitors attempting to perform fraudulent activities enabling you to identify them with confidence while providing a frictionless, positive experience for trusted visitors. 
```

Comment:
I know this wasn't the point of your update but maybe this could be better worded to something like this:

"[Smart Signals](https://dev.fingerprint.com/docs/smart-signals-overview) are an extensive set of actionable device intelligence signals that you can use to identify suspicious activity even when users are anonymous or visiting your site for the first time. By analyzing these signals and looking for unusual behavior you can make smarter decisions in real time to prevent fraudulent activities while still providing a positive experience for trusted visitors."

Reply:
Applied, thank you!